src/App.tsx

```diff
@@ -1,61 +1,41 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import todosFromServer from './api/todos';
+import { getUserById } from './services/user';
+import { Todo } from './types/Todo';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { TodoList } from './components/TodoList';
+
+const initialTodos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+function getNewTodoId(todos: Todo[]) {
```

Comment:
the code uses two types of function declaration, it is better to choose one so that the code is consistent